Software/app/freefall_lorawan/README.md

```diff
@@ -0,0 +1,76 @@
+2. Set `GNSE_ADVANCED_TRACER_ENALBE` to 0 in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h). This will make the device consume less power, but will also disable UART.
+3. Set `ACC_FF_ODR` in [app/conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h) to a lower value if the decreased performance of the free fall detection does not matter for your setup. [LIS2DH12.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/lib/LIS2DH12/LIS2DH12.h) in the LIS2DH12 library lists all the options for this.
+
+Additionally, try to remember that:
+- Setting the debugger will consume more power. The debugger is set in the macro `DEBUGGER_ON` in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h).
+- Keeping peripherals, such as the sensors, off by controlling the load switches will reduce power consumption significantly. But, of course, you can not turn the load switch connected to the accelerometer off without disabling the free fall detection as well.
+
+### LoRa extra's
+
+Some extra setup configurations can be done to change the behaviour of the LoRaWAN capabilities. These include:
+1. Setting the activation method (OTAA or ABP) in `LORAWAN_DEFAULT_ACTIVATION_TYPE` in [lora_app.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/lora_app.h). OTAA is recommended as it is the most secure.
+2. The data rate can be set in [lora_app.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/lora_app.h). The default configuration uses the ADR. Should you want to set your preferred data rate, set `LORAWAN_ADR_STATE` to `LORAMAC_HANDLER_ADR_OFF` and set `LORAWAN_DEFAULT_DATA_RATE` to your preference. A list of the options per region are shown in [Region.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/lib/STM32WLxx_LoRaWAN/LoRaWAN/Mac/region/Region.h) in the STM32WLxx_LoRaWAN library.
+3. `ACC_FF_LORA_PORT` can be changed in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h), which is used to configure to transmission port. The LoRaWAN keys mentioned in the default section can be altered here as well.
+4. Also in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h), the downlink port can be set by altering `ACC_FF_DOWNLINK_PORT`.
+
+### Debugger
+
+For debugging, the firmware has to support it first. The debugger is set in the macro `DEBUGGER_ON` in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h).
+
```

Comment:
This is pretty much the same in both files, so please look at the comments for the first one